myhpom/templates/myhpom/upload/index.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'myhpom/includes/upload_base.html' %}
 
 {% block widget_body %}
-<h3 class="h6">
+<h3 class="h5">
     Requirements for Advance Directive:
 </h3>
```

Comment:
This header (Requirements...) is the same color as the main `<h1>` for the dashboard.